Bitbucket.Authentication/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers by using the '*'
 // as shown below: [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Yes, it was just from my testing to allow me to be sure I'd update the lib. Feel free to keep it to 1.0.0